fixtures/completion/constant_with_namespace.php

```diff
@@ -9,14 +9,17 @@ function world() {
 
     }
 
-    \HELLO;
+    \HE
 }
 
 namespace {
 
+    /**
+     * Lorem ipsum dolor sit amet.
+     *
+     * @var bool
```

Comment:
I would not test the `@var` tag because I expect the type of a constant to be inferred